src/monodroid/jni/string-builder.h

```diff
@@ -0,0 +1,53 @@
+// Dear Emacs, this is a -*- C++ -*- header
+#ifndef __STRING_BUILDER_H
+#define __STRING_BUILDER_H
+
+#include <cstddef>
+#include <type_traits>
+#include <limits>
+
+namespace xamarin { namespace android { namespace internal {
+	class StringBuffer
```

Comment:
Do we need this class? It is only used as a base to another "empty" class.

Reply:
It's not complete yet, I've had it in this branch for a while and thought that it won't hurt to keep it around for further work, but perhaps it should be taken out for now.

Reply:
I've removed the class